Intersections_3/include/CGAL/Intersections_3/internal/intersection_3_1_impl.h

```diff
@@ -182,6 +209,73 @@ intersection(const typename K::Plane_3 &plane1,
     return intersection_return<typename K::Intersect_3, typename K::Plane_3, typename K::Plane_3>(plane1);
 }
 
+
+  //  triple plane intersection
+template <class K>
+boost::optional<typename K::Point_3>
+intersection_point(const typename K::Plane_3 &plane1,
+                   const typename K::Plane_3 &plane2,
+                   const typename K::Plane_3 &plane3,
+                   const K& k)
+{
+  typedef typename boost::optional<typename K::Point_3> result_type;
+   typedef typename K::FT FT;
+    const FT &m00 = plane1.a();
+    const FT &m01 = plane1.b();
+    const FT &m02 = plane1.c();
+    const FT &b0  = - plane1.d();
+    const FT &m10 = plane2.a();
+    const FT &m11 = plane2.b();
+    const FT &m12 = plane2.c();
+    const FT &b1  = - plane2.d();
+    const FT &m20 = plane3.a();
+    const FT &m21 = plane3.b();
+    const FT &m22 = plane3.c();
+    const FT &b2  = - plane3.d();
+    /*
+    const FT den = determinant(m00, m01, m02,
```

Comment:
if not garbage please add a comment saying that what is commented is what is done below more efficiently